src/main/java/com/capgemini/archaius/spring/ArchaiusBridgePropertyPlaceholderConfigurer.java

```diff
@@ -15,17 +15,17 @@
  */
 package com.capgemini.archaius.spring;
 
-import java.io.IOException;
-import java.util.Map;
-import java.util.Properties;
-
 import com.netflix.config.ConcurrentCompositeConfiguration;
 import org.apache.camel.spring.spi.BridgePropertyPlaceholderConfigurer;
 import org.apache.commons.configuration.ConfigurationConverter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.core.io.Resource;
 
+import java.io.IOException;

```

Comment:
Could you move these back up to the top? (Its a style thing - I'm guessing these moved down automatically based on your IDE-config.)